src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
That won't work without making making color to only use D65. i.e. Color<T> with each variant as Luma(Luma<D65, T>), Xyz(Xyz<D65,T>) etc